react/features/large-video/actions.js

```diff
@@ -21,22 +26,18 @@ export function selectParticipant() {
 
         if (conference) {
             const largeVideo = state['features/large-video'];
-            const tracks = state['features/base/tracks'];
-
             const id = largeVideo.participantId;
-            const videoTrack
-                = getTrackByMediaTypeAndParticipant(
-                    tracks,
-                    MEDIA_TYPE.VIDEO,
-                    id);
 
             try {
-                conference.selectParticipant(
-                    videoTrack && videoTrack.videoType === VIDEO_TYPE.CAMERA
-                        ? id
-                        : null);
+                conference.selectParticipant(id);
```

Comment:
I can't really remember why we had this logic here about VIDEO_TYPE.CAMERA. @virtuacoplenny can you please elaborate on it if you have an idea?

Reply:
There was a chat about this in the morning and the feedback I got is that it's a bug to do camera only.

Reply:
@yanas just some clarifications for posterity on why this is a bug; we always want to send the selected endpoint to the bridge, regardless of video media type, because the bridge does some special handling for the on-stage participant.

For example, the bridge may decide to never suspend the on-stage participant. Also now that screen-sharing simulcast is being rolled out with new Chrome releases, HD screen-sharing depends on that message reaching the bridge.

Reply:
I completely agree. That's why I was surprised to see we check for camera type.